src/main/java/seedu/address/model/card/Card.java

```diff
@@ -27,19 +27,22 @@
     private final UUID id;
     private final String front;
     private final String back;
+    private final Schedule schedule;
```

Comment:
either add issue to add tests, or add tests now

Reply:
I added tests